CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 Release Versions:
 
+- [5.1.0](#510)
```

Comment:
Is that because of the staging branch?

Reply:
Hmm no that is because it's missing on main. But maybe not the right place to put it then

Reply:
hmm I also didn't realize it was missing, up to you